### PR TITLE
[JS] chore: update setup for listBot / devOpsBot / twentyQuestions / orderBot

### PR DESCRIPTION
**Pull request by swatDong**
## Linked issues

closes: #976 #977 #978 #979

## Details

- listBot / devOpsBot / twentyQuestions / orderBot
  - Add `sample.env`
  - Remove redundant variables and yarn commands
  - Add steps to README
- devOpsBot
  - Add Teams Toolkit support (.vscode, .yml, ...)

## Attestation Checklist

- [X] My code follows the style guidelines of this project

- I have checked for/fixed spelling, linting, and other errors
- I have commented my code for clarity
- I have made corresponding changes to the documentation (we use [TypeDoc](https://typedoc.org/) to document our code)
- My changes generate no new warnings
- I have added tests that validates my changes, and provides sufficient test coverage. I have tested with:
  - Local testing
  - E2E testing in Teams
- New and existing unit tests pass locally with my changes

### Additional information

> Feel free to add other relevant information below

### CODE REVIEW

File: js/samples/06.assistants.b.orderBot/teamsapp.local.yml [hunk 1]
```diff
@@ -69,32 +69,10 @@ provision:
...
-      shell: powershell
-
   # Provides the Teams Toolkit .env file values to the apps runtime so they can be accessed with `process.env`.
   - uses: file/createOrUpdateEnvironmentFile
     with:
       target: ./.env
       envs:
         BOT_ID: ${{BOT_ID}}
         BOT_PASSWORD: ${{SECRET_BOT_PASSWORD}}
-        TEAMS_APP_TENANT_ID: ${{TEAMS_APP_TENANT_ID}}
```
Comment (lilyydu) [inline js/samples/06.assistants.b.orderBot/teamsapp.local.yml:96]: We can keep the tenant id here!

File: js/samples/04.e.twentyQuestions/README.md
Comment (lilyydu) [inline js/samples/04.e.twentyQuestions/README.md:1]: Could we move the table of contents underneath the description, above scaffolding?

File: js/samples/04.ai.e.chainedActions.devOpsBot/README.md [hunk 2]
```diff
@@ -2,6 +2,23 @@
 
 <!-- @import "[TOC]" {cmd="toc" depthFrom=1 depthTo=6 orderedList=false} -->
 
+<!-- code_chunk_output -->
```
Comment (lilyydu) [inline js/samples/04.ai.e.chainedActions.devOpsBot/README.md:5]: Move table of contents to below description

File: js/samples/04.ai.d.chainedActions.listBot/README.md
Comment (lilyydu) [inline js/samples/04.ai.d.chainedActions.listBot/README.md:1]: Move table of contents to below description

**Review by singhk97 — DISMISSED** [review]

File: js/samples/06.assistants.b.orderBot/teamsapp.local.yml (hunk 1 above)
Comment (swatDong) [reply js/samples/06.assistants.b.orderBot/teamsapp.local.yml:96]: Added back

File: js/samples/04.e.twentyQuestions/README.md
Comment (swatDong) [reply js/samples/04.e.twentyQuestions/README.md:1]: Updated.

File: js/samples/04.ai.d.chainedActions.listBot/README.md
Comment (swatDong) [reply js/samples/04.ai.d.chainedActions.listBot/README.md:1]: Updated

File: js/samples/04.ai.e.chainedActions.devOpsBot/README.md (hunk 2 above)
Comment (swatDong) [reply js/samples/04.ai.e.chainedActions.devOpsBot/README.md:7]: Updated

**Review by lilyydu — APPROVED** [review]

**Review by corinagum — APPROVED** [review]

**Pull request merged**